.ci/openshift-ci/happy-path-pod.yaml

```diff
@@ -12,6 +12,18 @@ spec:
       image: quay.io/eclipse/che-e2e:nightly
       imagePullPolicy: Always
       env:
+        - name: TS_SELENIUM_OCP_USERNAME
```

Comment:
this file shouldn't be committed at all in che-plugin-registry repository

Reply:
This file uses for setup plugins test.
For running tests on the openshift CI we need to configure a pod.
You can see that we get the "che-e2e" nightly test image
and define a list of specific variables necessary for
running exactly plugins test.

Reply:
but we can't get it from external location ?
because there is no link to che-plugin-registry in that file, it's a pure che e2e test config file